### DISCUSSION TITLE
cssbattle editor behaving weirdly

### DISCUSSION THREAD

**Discussion by prakash-niroula** [cssbattle/forum#289] [Bugs reports]
I don't know if this is only happening to me, [ I tried in incognito mode and changing account but same behavior ].
The cssbattle editor is behaving SO weird. I joined in like 3-4 days ago and it was fine and working perfect. But as I was trying to complete targets in cssbattle, I felt that the editor behaved differently. I mean when I write something and hit enter it moved my cursor to somewhere else and didn't add correctly indent.

## Things I felt weird
- When you write tags and press enter, no indentation is added [ which previously did ]
- When you are styling something and typing `{}` moves cursor to front [ which is totally weird, as it should move in between not the front ]
- Also when you press enter when cursor is between `{}`, It moves the cursor to the front and not in the between where code needs to be written and I have to manually move it to the section which is very irritating [ I don't think this is intentional ]
- When you press enter after defining a style example: `background: None;` and press enter, instead of going to the new line, cursor stays there but a new line is created, again I have to manually get to the position which is so irritating specially when working with a complicated target
- I am a newbie to this site and recently started using it. Everything was fine but as I tried doing today, It didn't work as it was supposed to be
- As much as I know, this shouldn't be an intentional feature so I reported it

**Comment by chinchang** [comment]
this is fixed now.
